### DISCUSSION TITLE
如何把封面的“论文题目”四个字的字号缩小到和题目一样大

### DISCUSSION THREAD

**Discussion by YunjiaXi** [sjtug/SJTUThesis#688] [提问] [answered]
被图书馆专家审核挑错了....

**Comment by skyzh** [comment, marked as answer]
最新版已经解决了这个问题。

<img width="859" alt="image" src="https://user-images.githubusercontent.com/4198311/121347970-5ed21d00-c95a-11eb-860e-efea78ef47a1.png">
